src/ResourceManager/Intune/Commands.Intune/ApiVersionHandler.cs

```diff
@@ -0,0 +1,95 @@
+﻿// ----------------------------------------------------------------------------------
```

Comment:
@rnikhilkumar Not in publically released cmdlets - customers only have access to official azure versions, it is ainappropriate to make test code available this way.